test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -60,6 +143,14 @@
 		}
 
 		cw.visitSource(className + ".java", null);
+
+		if (nestHost != null) {
+			cw.visitNestHost(nestHost);
+		}
+
+		if (valueUsedInCode != null) {
+			cw.visitNestHost(valueUsedInCode);
```

Comment:
This should be a call to `visitNestMember`.  I must have messed it up in the process of refactoring code.

Reply:
Fixed in commit fbba19114